respy/simulate.py

```diff
@@ -268,16 +273,26 @@ def _get_random_types(states, optim_paras, options):
     return types
 
 
-def _get_random_initial_experience(choice, optim_paras, options):
+def _get_random_initial_experience(states_df, choice, optim_paras, options):
```

Comment:
Do not alter this function now. This is done in #284. We are fine if you just change the ordering, sampling observables first then experiences. No need for more adjustments.